docs/building/netcoreapp3.0-instructions.md

```diff
@@ -0,0 +1,13 @@
+In order to build ML.NET for .NET Core 3.0, you need to do a few manual steps.
+
+1. Pick a version of the .NET Core 3.0 SDK you want to use. As of this writing, I'm using `3.0.100-alpha1-009622`. You can get the latest available version from the [dotnet/core-sdk README](https://github.com/dotnet/core-sdk#installers-and-binaries) page.
```

Comment:
>Pick [](start = 3, length = 4)

Thanks for adding this @eerhardt -- I found this impossibly confusing myself without asking you, I'm not sure how someone who didn't know the right person to ask would manage. :)